docs: BSR KMS Configuration Requirements (#3440)
hashicorp/boundary: website/content/docs/configuration/session-recording/create-storage-bucket.mdx

         }
      ]
   }

   ```

> - If KMS encryption is applied to the storage bucket, you will need to add these additional permissions for the Storage Bucket IAM User to the role policy:

Reviewer: ```suggestion
- If you apply KMS encryption to the storage bucket, you must add these additional permissions to the role policy for the storage bucket's IAM user:
```
